### PR TITLE
Force a save before taking screenshot in tooltip eyes tests

### PR DESCRIPTION
**Pull request by kozzi**
This forces the `.projected_updated_at` div to be consistent across screenshots and allow us to not have to exclude that region the UI tests.

FYI @ashercodeorg

### CODE REVIEW

**Comment by Bjvanminnen** [comment]
Thanks for doing this. This generally looks good, but I think I would prefer to see "I save the project" as a separate step from the eyes step, i.e.

```
Then I save the project
And I see no difference for "<identifier>"
```

File: dashboard/test/ui/step_definitions/applab.rb [hunk 1]
```diff
@@ -263,3 +263,11 @@ def get_scale_script
     };
   }
 end
+
+And /^I save the project$/ do
+  script = %Q{
+    Applab.serializeAndSave();
```
Comment (bcjordan) [inline dashboard/test/ui/step_definitions/applab.rb:269]: This is neat! I like how it pulls the JS out of the ruby code a bit :smile:

**Comment by kozzi** [comment]
With this merged, I also just removed the excluded region in the "tooltips for design mode" eyes test.

**Comment by Bjvanminnen** [comment]
awesome. thanks!

**Pull request merged**